Octokit/Clients/RepositoriesClient.cs

```diff
@@ -717,6 +725,53 @@ public Task<IReadOnlyList<RepositoryContributor>> GetAllContributors(long reposi
             return ApiConnection.GetAll<RepositoryContributor>(ApiUrls.RepositoryContributors(repositoryId), parameters, options);
         }
 
+        /// <summary>
+        /// Gets all topics for the specified repository.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://docs.github.com/en/rest/reference/repos#get-all-repository-topics">API documentation</a> for more details
+        /// </remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <returns>All topics associated with the repository.</returns>
+        [ManualRoute("GET", "/repos/{owner}/{repo}/topics")]
+        public async Task<IReadOnlyList<string>> GetAllTopics(string owner, string name)
+        {
+            Ensure.ArgumentNotNullOrEmptyString(owner, nameof(owner));
+            Ensure.ArgumentNotNullOrEmptyString(name, nameof(name));
+
+            var endpoint = ApiUrls.RepositoryTopics(owner, name);
+            var data = await ApiConnection.Get<List<string>>(endpoint).ConfigureAwait(false);
+
+            return data ?? new List<string>();
+        }
+
+        /// <summary>
+        /// Replaces all topics for the specified repository.
+        /// </summary>
+        /// <remarks>
+        /// See the <a href="https://docs.github.com/en/rest/reference/repos#replace-all-repository-topics">API documentation</a> for more details
+        ///
+        /// This is a replacement operation; it is not additive. To clear repository topics, for example, you could specify an empty list of topics here.
```

Comment:
❓ Do we want to include an overload, `ClearTopics`, that will call `ReplaceAllTopics` with an empty list of topics?

Reply:
I'd prefer to stick to the current API surface, as helpers that are specific to the .NET client may confuse users